libs/gretty-runner-jetty10/src/main/groovy/org/akhikhl/gretty/JettyConfigurerImpl.groovy

```diff
@@ -200,25 +199,12 @@ class JettyConfigurerImpl implements JettyConfigurer {
   def createWebAppContext(Map serverParams, Map webappParams) {
     List<String> webappClassPath = webappParams.webappClassPath
     JettyWebAppContext context = new JettyWebAppContext()
+    context.setThrowUnavailableOnStartupException(true)
```

Comment:
Doesn't Groovy allow this to be written as `context.throwUnavailableOnStartupException = true`?

Reply:
Yes, there is no semantic difference between the two.